src/types/index.d.ts

```diff
@@ -22,8 +22,8 @@ export interface ConstructorClientOptions {
 }
 
 export interface UserParameters {
-  sessionId?: Number;
-  clientId?: Number;
+  sessionId?: string;
```

Comment:
I think we send sessionId as number most cases. Is there a reason you changed this to string?